guiEditor/src/diagram/workbench.tsx

```diff
@@ -397,7 +434,28 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
             removeObservers();
         }, PointerEventTypes.POINTERUP);
 
-        scene.onPointerObservable.add(zoomFn, PointerEventTypes.POINTERWHEEL);
+
+        scene.onKeyboardObservable.add((k: KeyboardInfo, e: KeyboardEventTypes) => {
+            switch (k.event.key) {
+                case "q": //select?
+                    if (!this._forceSelecting)
+                        this.globalState.onSelectionObservable.notifyObservers();

```

Comment:
what is the reason for notifying void? is there a reason to do that on all observables? why not simply not notify?

Reply:
I might be missing some understanding as to what those observables are doing. When I see `onSelectionObservable` , i expect it to notify what was selected, and not only "something was selected".

Reply:
Sure, but it will be great to first fully understand what they actually do :-)